src/sql-parser/src/parser.rs

```diff
@@ -2877,15 +2877,18 @@ impl<'a> Parser<'a> {
         }))
     }
     fn parse_replica_option(&mut self) -> Result<ReplicaOption<Raw>, ParserError> {
-        let name = match self.expect_one_of_keywords(&[AVAILABILITY, REMOTE, SIZE])? {
-            AVAILABILITY => {
-                self.expect_keyword(ZONE)?;
-                ReplicaOptionName::AvailabilityZone
-            }
-            REMOTE => ReplicaOptionName::Remote,
-            SIZE => ReplicaOptionName::Size,
-            _ => unreachable!(),
-        };
+        let name =
```

Comment:
Can you add the new syntax to https://github.com/MaterializeInc/materialize/blob/main/src/sql-parser/tests/testdata/ddl?

Reply:
done